src/Assets/Plugins/WebGL/BugsnagAppTimings.jspre

```diff
@@ -0,0 +1,54 @@
+(function () {
```

Comment:
What imports/runs this file?

Reply:
It gets bundled in by emscripten – I wasn't sure if it would get concat'd or isolated, so thought I'd do the old school iife (with a semi-colon) just in case.